pkgs/development/python-modules/pyeclib/default.nix

```diff
@@ -1,14 +1,23 @@
-{ lib, stdenv, buildPythonPackage, fetchFromGitHub, liberasurecode, six }:
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, liberasurecode
+, setuptools
+, six
+, pytestCheckHook
+}:
 
 buildPythonPackage rec {
   pname = "pyeclib";
-  version = "unstable-2022-03-11";
+  version = "1.6.1";
+  pyproject = true;
```

Comment:
There is no pyproject file https://github.com/openstack/pyeclib/tree/1.6.1

Reply:
`pyproject = true` can (and might be preferred) be used with legacy projects, see https://github.com/NixOS/nixpkgs/issues/253154

Reply:
I don't think we need to jump ahead here until this becomes the default and then we don't need to set it.